### ISSUE TITLE
AppVeyor build fails due to addition of .NET Core 2.0.2 to build image

### ISSUE THREAD

**Issue by markekraus** [PowerShell/PowerShell#5201]
AppVeyor updated the Visual Studio 2017 image to include the 2.0.2 SDK on 2017-10-22
https://www.appveyor.com/updates/

This results in the build failing with the following:

```none
The currently installed .NET Command Line Tools is not the required version.
Installed version: 2.0.2
Required version: 2.0.0
Fix steps:
1. Remove the installed version from:
    - on windows '$env:LOCALAPPDATA\Microsoft\dotnet'
    - on macOS and linux '$env:HOME/.dotnet'
2. Run Start-PSBootstrap or Install-Dotnet
3. Start-PSBuild -Clea

Invoke-AppveyorTest
Run CoreCLR tests
CoreCLR pwsh.exe was not built
Invoke-AppveyorFinish
Packaging RID: 'win7-x64'; Packaging Configuration: 'Release'
Please ensure you have run 'Start-PSBuild -Clean -CrossGen -Runtime win7-x64 -Configuration Release'!
```

example build fail: https://ci.appveyor.com/project/PowerShell/powershell/build/6.0.0-beta.8-6270

**Comment by markekraus** [comment]
cc/ @TravisEz13

**Comment by iSazonov** [comment]
Interenting, the image contains 2.0.0 too? #5118 can help.

**Comment by markekraus** [comment]
I think it's because we are doing a `-ne` check on the version of `dotnet`:

https://github.com/PowerShell/PowerShell/blob/5b2bbcbf89251734e8557a55c5df40bfb1cba979/build.psm1#L414

**Comment by markekraus** [comment]
we would probably need to fix that and merge #5118

**Comment by markekraus** [comment]
It looks like #5118 was sufficient enough to fix the problem:

https://ci.appveyor.com/project/PowerShell/powershell/build/1.0.6275
